release template/CODE_OF_CONDUCT.md

```diff
@@ -68,7 +68,6 @@ members of the project's leadership.
 ## Attribution
 
 This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
-available at [http://contributor-covenant.org/version/1/4][version]
+available at <https://contributor-covenant.org/version/1/4>
```

Comment:
Uses the `<...>` notation (see [Markdown Guide](https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses)) to avoid having to repeat the URL twice.